backend_app/backend_app/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for backend_app project.
+
+Generated by 'django-admin startproject' using Django 4.2.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-l73(z@u+%n36!o9+k80$1t%ws2^b8=sy*6$qd25o1@+doc&@2('
```

Comment:
```suggestion
SECRET_KEY = (외부 값으로 대체)
```

서버의 비밀 키는 절대 외부로 공유되면 안되요.
[environ](https://www.daleseo.com/python-os-environ/) 같은 환경변수 라이브러리를 사용해서 외부에서 가져올 수 있게 수정하세요.